bin/git-coauthor

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
```

Comment:
We should check the number of the arguments.

Reply:
I agree, thanks 👍